homeassistant/components/lutron/__init__.py

```diff
@@ -69,6 +69,12 @@ def setup(hass, base_config):
                             button.button_type in ('SingleAction', 'Toggle')):
                         hass.data[LUTRON_DEVICES]['scene'].append(
                             (area.name, keypad.name, button, led))
+                # catch for main_repeater scenes
+                if (button.name != 'Unknown Button' and
+                        button.button_type in ('SingleAction', 'Toggle') and
+                        keypad.name == 'Enclosure Device 001'):
```

Comment:
Create a constante on top of the file i.e. as:
```
BUTTON_UNKNOWN = "Unknown Button"
BUTTON_TYPE = ('SingleAction', 'Toggle')
```
So you can use that also above.

Reply:
Resolved. Added 2 variables and replaced their usage throughout the file.

Reply:
Do you know for sure that the phantom scenes will always show up on a virtual keypad named `Enclosure Device 001`? The Lutron system can have two main repeaters and the name somewhat implies there might be more than one.

Reply:
@nickovs hm, I do not know this for sure, but the examples of xml dumps from ra2 configs online that I found all used this same naming convention, so I assumed it to be true.  FWIW, I do believe (from reading the hass lutron docs) that only a single main repeater is supported

Reply:
Unresolving this comment as comment by Pascal has not been addressed…